### ISSUE TITLE
Fixed gem version

### ISSUE THREAD

**Issue by khelll** [weppos/publicsuffix-ruby#97]
usually in gems it's set to:
 `gem.version = PublicSuffix::VERSION`
now it's set as 
`gem.version = '1.5.3'`
Is this on purpose?

**Comment by weppos** [comment]
The constant is there.
https://github.com/weppos/publicsuffix-ruby/blob/master/lib/public_suffix/version.rb#L20

I'm not sure I understood your request.

**Comment by khelll** [comment]
The constant is fixed in the gemspec file https://github.com/weppos/publicsuffix-ruby/blob/master/public_suffix.gemspec#L6

**Comment by weppos** [comment]
That's because the `gemfile` is generated by a Rake task.
https://github.com/weppos/publicsuffix-ruby/blob/master/Rakefile#L13

Why is that an issue to you?

**Comment by khelll** [comment]
Aha, interesting.
I was reading the documentation (which as of now belongs to version 2.0.0, while the latest version on rubygems is 1.5.3), and I was verifying why things wouldn't work for me locally, so I came by the code to check what is the latest version and I saw that line which misled me at first, since it shows 1.5.3 and that matches my local version. Now, things are clear, it seems you didn't upload the new version to Rubygems right?
